org.civicrm.reqangular/src/common/angular-date/main.js

```diff
@@ -32,4 +32,8 @@ Module.config(function($provide) {
     $provide.decorator('daypickerDirective', require('./src/decorators/DaypickerDirectiveDecorator'));
 
     $provide.decorator('datepickerPopupWrapDirective', require('./src/decorators/DatepickerPopupWrapDirectiveDecorator'));
-});
\ No newline at end of file
+});
+
+Module.run(function(DateFactory){
+    DateFactory.fetchDateFormatFromSettings();
```

Comment:
Is `DateFactory` is the right place to put methods concerned with getting the current date format? Shouldn't the concern of a factory be only to create new object, rather than additionally provide information about the current state of an application?

If we have already this `angular-date` module which is a collection of controllers/services/directives related to the date, why not make a specific service whose job is only to retrieve the current date format?